db/data_migration/20150224084313_backfill_political_flag_on_editions.rb

```diff
@@ -0,0 +1,92 @@
+# As per https://docs.google.com/a/digital.cabinet-office.gov.uk/spreadsheets/d/1412V79VMtpfZWNMYJhV7A2QFDKvUdseRtnA35bm-gmY/edit#gid=1194852478
+#
+# Exceptions at time of writing:
+# - Blog posts are not stored in Whitehall
+# - Annual reports are covered under corporate reports
+# - Government responses, press releases and news stories are covered under news articles
+
+POLITICAL_PUBLICATION_TYPES = [
+  PublicationType::ImpactAssessment,
+  PublicationType::InternationalTreaty,
+  PublicationType::PolicyPaper,
+  PublicationType::ResearchAndAnalysis,
+  PublicationType::CorporateReport
+].freeze
+
+POLITICAL_ORGS = %w[
+  attorney-generals-office
+  cabinet-office
+  department-for-business-enterprise-and-regulatory-reform
+  department-for-business-innovation-skills
+  department-for-children-schools-and-families
+  department-for-communities-and-local-government
+  department-for-constitutional-affairs
+  department-for-culture-media-sport
+  department-for-education
+  department-for-education-and-skills
+  department-for-environment-food-rural-affairs
+  department-for-innovation-universities-and-skills
+  department-for-international-development
+  department-for-transport
+  department-for-work-pensions
+  department-of-constitutional-affairs
+  department-of-energy-climate-change
+  department-of-health
+  department-of-inland-revenue
+  department-of-national-heritage
+  department-of-social-security
+  department-of-the-environment-transport-and-the-regions
+  department-of-trade-and-industry
+  exchequer-and-audit-department
+  foreign-commonwealth-office
+  hm-treasury
+  home-office
+  law-officers-departments
+  lord-chancellors-department
+  ministry-of-defence
+  ministry-of-justice
+  northern-ireland-office
+  office-of-the-advocate-general-for-scotland
+  the-office-of-the-leader-of-the-house-of-commons
+  office-of-the-leader-of-the-house-of-lords
+  prime-ministers-office-10-downing-street
+  scotland-office
+  scottish-office
+  uk-export-finance
+  wales-office
+].freeze
```

Comment:
It is already there. Just not with any data in it at the moment: https://github.com/alphagov/whitehall/pull/1978